packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -175,10 +175,11 @@ const ActiveVisitsTable = () => {
                           <th colSpan={headers.length + 2}>
                             <ExtensionSlot
                               className={styles.visitSummaryContainer}
-                              extensionSlotName="visit-summary-slot"
+                              extensionSlotName="visit-detail-overview-slot"
```

Comment:
Yeah this seems like a worse name

Reply:
I'm not exactly sure what we're putting in this slot so hard for me to recommend a different name.

Reply:
let's just call this the av-visit-overview-slot for now. @brandones do we need to start a convention to name-space slots? Calling this the "visit-overview-slot" could eventually be used somewhere else which would be problematic. calling this the active-visits-widget-visit-overview-slow seems a tad long.

Reply:
Oh that's an interesting question. Right now slot names do need to be globally unique. I would rather we lift that restriction and make them only have to be unique per module rather than establish a namespacing convention.

For this one, I would advocate calling it `visit-row-details-slot`. [A slot is a place](http://o3-dev.docs.openmrs.org/#/main/extensions?id=naming-slots).